src/pages/api/auth/[...nextauth].ts

```diff
@@ -21,4 +21,23 @@ export default NextAuth({
   pages: {
     signIn: '/auth/signin',
   },
+  session: {
+    strategy: 'jwt',
+  },
+  callbacks: {
+    async session({ session, token, user }) {
+      session.user.id = token.id;
+      session.accessToken = token.accessToken;
+      return session;
+    },
+    async jwt({ token, user, account, profile, isNewUser }) {
+      if (user) {
+        token.id = user.id;
+      }
+      if (account) {
```

Comment:
idt we need the accesstoken on the jwt token either, unless we're checking if the access_token is valid during the jwt token validation with `getToken()`

Reply:
ngl I took these two sections out and stuff broke soooo

Reply:
hmm... when it worked when I tested on your branch. Maybe delete ur mongodb collection and try again?